eng/helix/content/RunTests/TestRunner.cs

```diff
@@ -341,9 +347,7 @@ public void UploadResults()
                     // Combine the directory name + log name for the copied log file name to avoid overwriting duplicate test names in different test projects
                     var logName = $"{Path.GetFileName(Path.GetDirectoryName(file))}_{Path.GetFileName(file)}";
                     Console.WriteLine($"Copying: {file} to {Path.Combine(HELIX_WORKITEM_UPLOAD_ROOT, logName)}");
-                    // Need to copy to HELIX_WORKITEM_UPLOAD_ROOT and HELIX_WORKITEM_UPLOAD_ROOT/../ in order for Azure Devops attachments to link properly and for Helix to store the logs
                     File.Copy(file, Path.Combine(HELIX_WORKITEM_UPLOAD_ROOT, logName));
-                    File.Copy(file, Path.Combine(HELIX_WORKITEM_UPLOAD_ROOT, "..", logName));
```

Comment:
This and the changes below in this file look like cleanup. Did something change in the Helix underpinnings❔

Reply:
This was cleanup that got lost in a really old PR that didn't get merged, they fixed the bug where we had to copy to both upload directories a long time ago